app/controllers/planning_applications/preapplication_services_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module PlanningApplications
+  class PreapplicationServicesController < AuthenticationController
```

Comment:
If we change the name to `AdditionalService` for the model then we should change this as well